BlockSettleSigner/main.cpp

```diff
@@ -336,7 +336,7 @@ static int QMLApp(int argc, char **argv
 
          terminalConnectionTimer.setSingleShot(true);
          //BST-2786
-         terminalConnectionTimer.setInterval(std::chrono::milliseconds{ 5000 });
+         terminalConnectionTimer.setInterval(std::chrono::milliseconds{ 25000 });
```

Comment:
Is this intended?

Reply:
Yes, this helps to run signer properly with 2 wallets with 3k+ addresses in Windows debug. Lower delay values lead to signer TCP error. Scott was also complaining about that on Linux release with D8s prod wallet.

Reply:
I think that we are going a bit wrong path. What if there is even bigger wallet and slower PC? This must be a timer for establishing network connection, not for a complete loading of wallets. Just thoughts.

Reply:
Agree, I wasn't trying to fix it once and for all - this is just a workaround for an average comfort use. We can return to fixing it at signer arch refactoring (with bus and adapters).